checkov/terraform/checks/resource/aws/LambdaCodeSigningConfigured.py

```diff
@@ -0,0 +1,21 @@
+from checkov.common.models.consts import ANY_VALUE
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+
+
+class LambdaCodeSigningConfigured(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure that AWS Lambda function is configured for a code-signing with signing profiles, which define the trusted publishers for this function"
+        id = "CKV_AWS_272"
+        supported_resources = ['aws_lambda_function']
+        categories = [CheckCategories.GENERAL_SECURITY]
```

Comment:
```suggestion
        categories = [CheckCategories.SUPPLY_CHAIN]
```
I'm not sure, but I think supply chain fits here better. @tsmithv11 any thoughts about it?

Reply:
I agree, we're trying to avoid `GENERAL SECURITY` and `SUPPLY_CHAIN` makes sense